src/crab_utils/debug.cpp

```diff
@@ -8,6 +8,6 @@ std::set<std::string> CrabLog;
 
 unsigned CrabVerbosity = 0;
 
-bool CrabWarningFlag = true;
+bool CrabWarningFlag = false;
```

Comment:
why this change?

Reply:
I find these warnings not useful most of the time - they mostly warn about values becoming TOP, which does not hurt soundness, and it clutters the output of the tests. 
(BTW I've just pushed this change in #245. Of course it can be reverted).